src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -48,6 +48,8 @@ void listDescribeReflectCreateDelete() throws IOException {
 
     // check that the name and notebook name match
     assertEquals(name, createdNotebook.name, "create output matches name");
+    assertTrue(
+        createdNotebook.metadata.toString().contains("foo=bar"), "create output matches metadata");
```

Comment:
You don't need to convert the map to a string. Do a `get()` on it like `assertEquals("bar", createdNotebook.metadata.get("foo"))`